test/motd.ts

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) 2022, Compiler Explorer Authors
```

Comment:
Would love some help on how to set this up. I can't add them to `static/test` because it complains about sion stuff, and I can't enable it here either because it complains about wrong syntax (as it probably can't understand ts?)

Reply:
Oh gosh. Yeah getting some UI testing would be amazing. We will probably have to do some magic with `tsc` somehow. I can have a poke about too.